lib/ansible/modules/network/meraki/meraki_config_template.py

```diff
@@ -83,7 +83,10 @@
 def get_config_templates(meraki, org_id):
     path = meraki.construct_path('get_all', org_id=org_id)
     response = meraki.request(path, 'GET')
-    return response
+    if meraki.status == 200:
+        return response
+    else:
```

Comment:
Again, the else here is unneeded and unwanted.

Reply:
Looking at this, I would prefer even this:
```python
if meraki.status != 200:
    meraki.fail_json(msg='Unable to get configuration templates')

return response
```

So that the default flow returns the response, except if the status is not 200. I think that's more clear, exceptions go into if-blocks, default flow is to succeed.